app/serializers/user_serializer.rb

```diff
@@ -89,6 +89,9 @@ def services
     service_list << BackendServices::USER_PROFILE if object.can_access_user_profile?
     service_list << BackendServices::APPEALS_STATUS if beta_enabled?(object.uuid, 'appeals_status') &&
                                                        object.can_access_appeals?
+
+    service_list << BackendServices::SAVE_IN_PROGRESS if object.can_save_partial_forms?
```

Comment:
`service_list` represents features within Vets.gov that a user has access to.  `vets-website` consumes this list as a means to either block or allow a given user into a certain area of the Vets.gov site.

`BackendServices::SAVE_IN_PROGRESS` and `BackendServices::FORM_PREFILL`  _mostly_ fits that description but I wanted to point out it's kind of like apples to pears.  That said, I'm OK with it.